kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/AbstractServicesUi.java

```diff
@@ -607,7 +608,8 @@ public void onChange(ChangeEvent event) {
                     }
                 } catch (NumberFormatException e) {
                     this.valid.put(param.getId(), false);
-                    result.add(new BasicEditorError(box, box.getText(), e.getLocalizedMessage()));
+                    result.add(new BasicEditorError(box, box.getText(),
```

Comment:
It might be better to check if the value is a valid numeric and then adjust the error message accordingly.